docs/api/authentication.md

```diff
@@ -20,6 +21,9 @@ $ ssh git@git-server.com git-lfs-authenticate foo/bar.git download
     "Authorization": "RemoteAuth some-token"
   },
 
+  # optional, for expiring tokens, preferred over expires_at
+  "expires_in": 86400000000000
```

Comment:
Super minor, but the example values in this PR still show the values in nanoseconds :)